natgeo/mortar
# [SYTO-808] Fixing code broken during a poor merge conflict resolution session

@bpearsoncse This is the fix for the merge conflict I fixed incorrectly, if you can take a look I'd appreciate it.

To test:
- Pull feature/SYTO-808-fix-merge-fix
- Run `npm test` to verify tests still pass
- Run `gulp serve`
- Verify your PromoText component is behaving properly again
